userprofile/templates/userprofile/login.html

```diff
@@ -20,6 +20,29 @@ <h1>{% blocktrans %}Log in to {{ brand_name }}{% endblocktrans %}</h1>
 	<div id="login-box-row" class="row">
 
 		{% if shibboleth_login %}
+			<section aria-labelledby="login-shibboleth-users" class="col-sm-6 col-md-3 login-box">
+				<div class="login-section">
+					{% if shibboleth_title_text %}
+						<h2 id="login-haka-users">
```

Comment:
This `<h2>` has the wrong id. This element is referred to by the section's `aria-labelledby` attribute. The ARIA attributes are important for accessibility. This A+ page was rewritten for accessibility last year so let's keep it accessible.

Change `login-haka-users` to `login-shibboleth-users`.

Reply:
good catch, will do